build.gradle

```diff
@@ -22,7 +22,7 @@ dependencies {
 	testImplementation group: 'net.runelite', name:'client', version: runeLiteVersion
 	testImplementation group: 'net.runelite', name:'jshell', version: runeLiteVersion
 
-    testImplementation 'org.projectlombok:lombok:1.18.4'
+	testImplementation 'org.projectlombok:lombok:1.18.4'
```

Comment:
Nothing is changed here; unsure of why it's showing like this in the diff. Shows aligned properly in-editor.